src/content/docs/validator/external-chains/xrpl-evm.mdx

```diff
@@ -210,25 +210,42 @@ INF finalized block block_app_hash=... height=13242560 ...
 - If you receive an error that says `Error: set min gas price in app.toml or flag or env variable: error in app.toml`. Set the `minimum-gas-prices` field in `app.toml` to a value for example `0.25axrp`
 
 ### Ampd Configuration
-Once your node is up and running, you need to add the XRPL-EVM chain to your ampd's `config.toml`file. This can be done by adding in the following configuration:
+Once your node is up and running, you need to add the XRPL-EVM chain to your [ampd daemon's](https://github.com/axelarnetwork/axelar-amplifier/tree/main/ampd) `config.toml`file. This can be done by adding in the following configuration:
 
 For testnet:
 
-```toml
-  [[handlers]]
-  chain_finalization="RPCFinalizedBlock"
-  chain_name="xrpl-evm"
-  chain_rpc_url="https://rpc.testnet.xrplevm.org"
-  cosmwasm_contract="axelar1q8kn9t39ddpce42atk0d6wpdudr6djqxmz689m3nxy92ck0nnftqxfsuyk"
-  type="EvmMsgVerifier"
-
-  [[handlers]]
-  chain_finalization="RPCFinalizedBlock"
-  chain_name="xrpl-evm"
-  chain_rpc_url="https://rpc.testnet.xrplevm.org"
-  cosmwasm_contract="axelar1q8kn9t39ddpce42atk0d6wpdudr6djqxmz689m3nxy92ck0nnftqxfsuyk"
-  type="EvmVerifierSetVerifier"
+<tabs>
+    <tab-item title="Mainnet" class="bleed">
+    ```toml
+      [[handlers]]
+      type = 'MultisigSigner'
```

Comment:
verifier handler is missing

Reply:
since xrpl-evm isn't live on mainnet, only testnet section should be kept

Reply:
Okay will just keep this for testnet and remove mainnet (leaving a note in doc that it will be added once we move forward with integration)

```toml
  [[handlers]]
  type="EvmMsgVerifier"
  chain_finalization="RPCFinalizedBlock"
  chain_name="xrpl-evm"
  chain_rpc_url="your_own_rpc"
  cosmwasm_contract="axelar1q8kn9t39ddpce42atk0d6wpdudr6djqxmz689m3nxy92ck0nnftqxfsuyk"

  [[handlers]]
  type="EvmVerifierSetVerifier"
  chain_finalization="RPCFinalizedBlock"
  chain_name="xrpl-evm"
  chain_rpc_url="your_own_rpc"
  cosmwasm_contract="axelar1q8kn9t39ddpce42atk0d6wpdudr6djqxmz689m3nxy92ck0nnftqxfsuyk"

  [[handlers]]
  type = 'MultisigSigner'
  chain_name = 'xrpl-evm'
  cosmwasm_contract = 'axelar14a4ar5jh7ue4wg28jwsspf23r8k68j7g5d6d3fsttrhp42ajn4xq6zayy5'
```